### PR TITLE
[FIX] prevent singleton error when archiving multiple products at a time

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

**Comment by Cedric-Pigeon** [comment]
@sebastienbeau

**Comment by sebastienbeau** [comment]
It's ok for me! let's wait travis

**Review by sebastienbeau — APPROVED** [review]

File: shopinvader/models/shopinvader_product.py [hunk 1]
```diff
@@ -38,9 +38,9 @@ class ShopinvaderProduct(models.Model):
 
     @api.multi
     def _inverse_active(self):
-        inactive = self.filtered(lambda p: not p.active)
-        inactive.shopinvader_variant_ids.write(
-            {'active': False})
+        for inactive in self.filtered(lambda p: not p.active):
```
Comment (lmignon) [inline shopinvader/models/shopinvader_product.py]: @Cedric-Pigeon Why not `self.filtered(lambda p: not p.active).mapped('shopinvader_variant_ids').write( ....)`? To write all variants at once and reduce the number of sql queries

**Comment by Cedric-Pigeon** [comment]
@lmignon Indeed, code updated !

**Review by lmignon — APPROVED** [review]

**Comment by Cedric-Pigeon** [comment]
@sebastienbeau Any idea why Travis failed ?

**Comment by sebastienbeau** [comment]
I think travis was red due to random issue with cache : https://github.com/akretion/odoo-shopinvader/pull/124

**Pull request merged**